### PR TITLE
feat: proto `lint` and `format` tooling

### PR DESCRIPTION
**Pull request by aljo242**
Add:
- `make proto-lint`
- `make proto-format`
- `make proto-breaking-check`

Run:
- `make proto-format` and commit
- `make proto-lint` and make some fixes
- regenerate proto

The only breaking change is that there was a file in `participationrewards` that assigned mistakenly to the `interchainstaking` package.  This has been fixed

NOTE:
- only other changes to `*.pb.go` files should be modifying the file descriptor

### CODE REVIEW

File: proto/quicksilver/interchainstaking/v1/proposals.proto [hunk 1]
```diff
@@ -32,59 +30,51 @@ message RegisterZoneProposalWithDeposit {
...
+  string connection_id = 3 [(gogoproto.moretags) = "yaml:\"connection_id\""];
+  string base_denom = 4 [(gogoproto.moretags) = "yaml:\"base_denom\""];
+  string local_denom = 5 [(gogoproto.moretags) = "yaml:\"local_denom\""];
+  string account_prefix = 6 [(gogoproto.moretags) = "yaml:\"account_prefix\""];
+  bool multi_send = 7 [(gogoproto.moretags) = "yaml:\"multi_send\""];
+  bool liquidity_module = 8 [(gogoproto.moretags) = "yaml:\"liquidity_module\""];
+  string deposit = 9 [(gogoproto.moretags) = "yaml:\"deposit\""];
+  bool return_to_sender = 10 [(gogoproto.moretags) = "yaml:\"return_to_sender\""];
+  bool deposits_enabled = 11 [(gogoproto.moretags) = "yaml:\"deposits_enabled\""];
+  bool unbonding_enabled = 12 [(gogoproto.moretags) = "yaml:\"deposits_enabled\""];
```
Comment (joe-bowman) [inline proto/quicksilver/interchainstaking/v1/proposals.proto:45]:
```suggestion
  bool unbonding_enabled = 12 [(gogoproto.moretags) = "yaml:\"unbonding_enabled\""];
```

hasn't caused us issues before but lets fix this now before it bites us in the ass later!

**Review by ajansari95 — APPROVED** [review]
lgtm

**Review by joe-bowman — COMMENTED** [review]
lgtm

**Review by joe-bowman — APPROVED** [review]
lgtm

**Pull request merged**